src/screens/ConfigModeScreen.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+#include "DisplayController.h"
+
+class ConfigModeScreen {
```

Comment:
Would it make sense for the captivePortal to be setup and managed in this class?

Reply:
Same thing as above, I think declaring every possible thing at the top level (because this whole codebase is pretty small) will keep other parts clean.